packages/dds/tree/api-report/tree.public.api.md

```diff
@@ -431,7 +431,7 @@ export interface TreeNodeApi {
     parent(node: TreeNode): TreeNode | undefined;
     schema<T extends TreeNode | TreeLeafValue>(node: T): TreeNodeSchema<string, NodeKind, unknown, T>;
     shortId(node: TreeNode): number | string | undefined;
-    readonly status: (node: TreeNode) => TreeStatus;
+    status(node: TreeNode): TreeStatus;
```

Comment:
I don't expect this to have a meaningful difference to users of the API. I expected that the use of a readonly arrow function property rather than an inline function was a mistake, because it is at odds with the rest of the functions here.